paper-fab-speed-dial.html

```diff
@@ -103,6 +111,12 @@
 
 			this.opened = false;
 		},
+		
+		// Protected methods
+		_computeIcon: function(icon, src) {
+			// If we don't have an icon and we don't have a src then we default to the 'add' icon
+			return ((icon === undefined || icon === '') && (src === undefined || src === '')) ? 'add' : undefined;
```

Comment:
Please also add tests for this function: https://github.com/Collaborne/paper-fab-speed-dial/blob/master/test/paper-fab-speed-dial.html